source/js/components/join/language-select.jsx

```diff
@@ -38,7 +38,7 @@ export default class LanguageSelect extends React.Component {
     return (
       <select
         value={this.props.selectedLang}
-        onChange={evt => this.handleChange(evt)}
+        onBlur={evt => this.handleChange(evt)}
```

Comment:
You know what, I read up the specs again, and I think the ESLint rule here is plain wrong: `onChange` *always* fires, there is literally no reason to _replace_ it with `onBlur`, because blurring already triggers the `change` event as part of the blur.

Reply:
turns out this was entirely the wrong place to encode `value`